src/main/java/com/aws/greengrass/localdebugconsole/SimpleHttpServer.java

```diff
@@ -100,35 +127,72 @@ public SimpleHttpServer(Topics t, Kernel kernel, DeviceConfiguration deviceConfi
     public void postInject() {
         super.postInject();
         config.lookup(CONFIGURATION_CONFIG_KEY, "port").dflt(port).subscribe((w, n) -> {
+            int oldPort = port;
             port = Coerce.toInt(n);
             if (port < 1000) {
                 logger.atWarn().kv("port", port).kv("defaultPort", DEFAULT_HTTP_PORT)
                         .log("Port number should not be smaller than 1000. Using default.");
                 port = DEFAULT_HTTP_PORT;
             }
```

Comment:
this should be < 1024 - I should have caught this yesterday

Reply:
Fixed